src/index.js

```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import * as serviceWorker from '../../react-mipt-course-homeworks/src/serviceWorker';
```

Comment:
Here is the problem: if you clone the project in any directory except `react-mipt-course-homeworks` it will fail (I clone homeworks with login prefix to know who is the owner)